# Case Study-Titan Insurance Company-

The Titan Insurance Company has just installed a new incentive payment scheme for its lift policy sales force. It wants to have an early view of the success or failure of the new scheme. Indications are that the sales force is selling more policies, but sales always vary in an unpredictable pattern from month to month and it is not clear that the scheme has made a significant difference.

Life Insurance companies typically measure the monthly output of a salesperson as the total sum assured for the policies sold by that person during the month. For example, suppose salesperson X has, in the month, sold seven policies for which the sums assured are £1000, £2500, £3000, £5000, £10000, £35000. X's output for the month is the total of these sums assured, £61,500. Titan's new scheme is that the sales force receives low regular salaries but are paid large bonuses related to their output (i.e. to the total sum assured of policies sold by them). The scheme is expensive for the company, but they are looking for sales increases which more than compensate. The agreement with the sales force is that if the scheme does not at least break even for the company, it will be abandoned after six months.

The scheme has now been in operation for four months. It has settled down after fluctuations in the first two months due to the changeover.

To test the effectiveness of the scheme, Titan have taken a random sample of 30 salespeople measured their output in the penultimate month prior to changeover and then measured it in the fourth month after the changeover (they have deliberately chosen months not too close to the changeover). 
 
 
 3) What conclusion does the test (p-value) lead to? (2.5 points)
 
 
 4) Suppose it has been calculated that in order for Titan to break even, the average output must increase by £5000 in the scheme compared to the old scheme. If this figure is alternative hypothesis, what is:
    
    i) The probability of a type 1 error? (2.5 points)
     
    ii) What is the p- value of the hypothesis test if we test for a difference of £5000? (10 points)
    
    iii) Power of the test (5 points)


In [1]:
import pandas as pd
from scipy.stats import t, ttest_rel, ttest_1samp, ttest_ind, mannwhitneyu, levene, shapiro, f_oneway
from statsmodels.stats.power import ttest_power
import matplotlib.pyplot as plt


## Question 1 

Find the mean of old scheme and new scheme column. (5 points)

In [2]:
#Data Preparation

TitanInsurance_OldScheme = pd.DataFrame([57,103,59,75,84,73,35,110,44,82,67,64,78,53,41,39,80,87,73,65,28,62,49,84,63,77,67,101,91,50])
TitanInsurance_NewScheme = pd.DataFrame([62,122,54,82,84,86,32,104,38,107,84,85,99,39,34,58,73,53,66,78,41,71,38,95,81,58,75,94,100,68])

#Since given data are in thousands, it will be better to convert them in thousands.
TitanInsurance_OldScheme = TitanInsurance_OldScheme*1000
TitanInsurance_NewScheme = TitanInsurance_NewScheme*1000



In [3]:
#Find the mean of TitanInsurance_OldScheme
TitanInsurance_OldScheme.describe()

0
count      30.000000
mean    68033.333333
std     20455.980212
min     28000.000000
25%     54000.000000
50%     67000.000000
75%     81500.000000
max    110000.000000

In [4]:
#Find the mean of TitanInsurance_NewScheme
TitanInsurance_NewScheme.describe()

0
count      30.000000
mean    72033.333333
std     24062.394947
min     32000.000000
25%     55000.000000
50%     74000.000000
75%     85750.000000
max    122000.000000

In [5]:
from numpy import mean
mean1 = mean(TitanInsurance_NewScheme)
print (mean1)


0    72033.333333
dtype: float64


Mean of the oldscheme = 68033.333333
Mean of the newscheme = 72033.333333

## Question 2

Use the five percent significance test over the data to determine the p value to check new scheme has significantly raised outputs? (10 points)

In [6]:
# compare samples
#Null Hypothesis: There is no difference of output between old and new scheme
#Alternate Hypothesis: There is a difference of output between old and new scheme

stat, p =ttest_rel(TitanInsurance_OldScheme, TitanInsurance_NewScheme)
print('Statistics=%.3f, p=%.3f' % (stat, p))
# interpret
alpha = 0.05
if p > alpha:
    print('Same distributions (fail to reject H0)')
else:
    print('Different distributions (reject H0)')

Statistics=-1.556, p=0.131
Same distributions (fail to reject H0)


Since p-value (=0.131) is higher than 0.05, we fail to reject NULL hypothesis. The new scheme has NOT significantly raised outputs.

## Question 3

3) What conclusion does the test (p-value) lead to? (2.5 points)

Since p-value (=0.131) is higher than 0.05, we accept (fail to reject) NULL hypothesis. The new scheme has NOT significantly raised outputs.

## Question 4

4) Suppose it has been calculated that in order for Titan to break even, the average output must increase by £5000 in the scheme compared to the old scheme. If this figure is alternative hypothesis, what is:

i) The probability of a type 1 error? (2.5 points)

ii) What is the p- value of the hypothesis test if we test for a difference of £5000? (10 points)

iii) Power of the test (5 points)


In [16]:
#Null Hypothesis: Average output has not increased by £5000 in the new scheme compared to the old scheme (mean difference < 5000)
#Alternate Hypothesis: Average output has increased by £5000 in the new scheme compared to the old scheme (mean difference > 5000)

#Let  μ2 = Average sums assured by salesperson AFTER changeover.

#μ1 = Average sums assured by salesperson BEFORE changeover.

#μd = μ2 – μ1   
#H0: μd ≤ 5000  
#HA: μd > 5000

μ2 = mean(TitanInsurance_NewScheme)
μ1 = mean(TitanInsurance_OldScheme)
μd = μ2 - μ1

MiuDiff = TitanInsurance_NewScheme-TitanInsurance_OldScheme

stat1, p1 = ttest_1samp(MiuDiff,5000)
print('Statistics1=%.3f, p1=%.3f' % (stat1, p1))
# interpret
alpha = 0.05
if p1 > alpha:
    print('Same distributions (fail to reject H0)')
else:
    print('Different distributions (reject H0)')


#Probability of Type I error = significant level = 0.05 or 5%
TypeI_Error = 1 - 0.14
print ("i) Probability of making Type I error is ", TypeI_Error)

print ("ii) the p- value of the hypothesis test if we test for a difference of £5000 ", p1)



Statistics1=-0.389, p1=0.700
Same distributions (fail to reject H0)
i) Probability of making Type I error is  0.86
ii) the p- value of the hypothesis test if we test for a difference of £5000  [0.70013349]


In [23]:
# Calculating Power of Test
# Compute the difference in Means between Population and Sample and divide it by Standard Deviation
# Set the number of Observations
# Set the alpha value and alternative values 'two-sided' , 'larger' , 'smaller'
import numpy as np
dif=np.mean(MiuDiff)/np.std(MiuDiff)
print (dif)

0    0.288926
dtype: float64


In [21]:
ptest=ttest_power(dif, nobs=30, alpha=0.86, alternative='two-sided')
print ("iii) Power of the test is ", ptest)

iii) Power of the test is  0    0.95943
dtype: float64
